machine_learning_hep/data/database_ml_parameters_Dspp.yml

```diff
@@ -244,47 +244,46 @@ Dspp:
       data: [0.2,0.2,0.2,0.2] #list of nbins
       mc: [0.2,0.2,0.2,0.2] #list of nbins
 
-    probcutoptimal: [0.4,0.4,0.3,0.3] #list of nbins
+    probcutoptimal: [0.5,0.5,0.4,0.3] #list of nbins
     
-    sel_an_binmin: [2,3,4,5,6,8,12]
-    sel_an_binmax: [3,4,5,6,8,12,24]
-    binning_matching: [0,0,1,1,1,2,3]  
+    sel_an_binmin: [2,4,6,8,12,16]
+    sel_an_binmax: [4,6,8,12,16,24]
+    binning_matching: [0,1,1,2,3,3]  
     sel_final_fineptbins: ["is_ev_rej==0",
                            "is_ev_rej==0", 
                            "is_ev_rej==0",
                            "is_ev_rej==0",
                            "is_ev_rej==0",
-                           "is_ev_rej==0",
                            "is_ev_rej==0"]  
     
-    mass_fit_lim: [1.90, 2.05] # region for the fit of the invariant mass distribution [GeV/c^2]
+    mass_fit_lim: [1.75, 2.15] # region for the fit of the invariant mass distribution [GeV/c^2]
     bin_width: 0.001 # bin width of the invariant mass histogram
     presel_gen_eff: "abs(y_cand) < 0.5 and abs(z_vtx_gen) < 10"
     usefit: true
-    sgnfunc: [kGaus,kGaus,kGaus,kGaus,kGaus,kGaus,kGaus]
-    bkgfunc: [Pol2,Pol2,Pol2,Pol2,Pol2,Pol2,Pol2]
+    sgnfunc: [kGaus,kGaus,kGaus,kGaus,kGaus,kGaus,kGaus,kGaus]
+    bkgfunc: [Pol2,Pol2,Pol2,Pol2,Pol2,Pol2,Pol2,Pol2]
     masspeak: 1.969
-    massmin: [1.90,1.90,1.90,1.90,1.90,1.90,1.90]
-    massmax: [2.05,2.05,2.05,2.05,2.05,2.05,2.05]
-    rebin: [6,6,6,6,6,6,6]
-    includesecpeak: [0,0,0,0,0,0,0]
+    massmin: [1.75,1.75,1.75,1.75,1.75,1.75]
+    massmax: [2.15,2.15,2.15,2.15,2.15,2.15]
+    rebin: [2,2,3,3,4,4]
+    includesecpeak: [1,1,1,1,1,1]
     masssecpeak: 1.969
-    FixedMean: True
+    FixedMean: false
     SetFixGaussianSigma: true
     SetInitialGaussianMean: true
     dolikelihood: true
-    sigmaarray: [0.01,0.01,0.01,0.01,0.01,0.01,0.01]
-    FixedSigma: true 
+    sigmaarray: [0.00637,0.006688,0.00715,0.008,0.00993,0.011]
+    FixedSigma: false
     fitcase: Ds
-    latexnamemeson: "D^{0}"
+    latexnamemeson: "D_{s}"
     latexbin2var: "n_{trkl}"
     nevents: 1700000000.
     dodoublecross: false
 
   validation:
     data:
-      dir: [dataval] #list of periods
-      dirmerged: datavaltot
+      dir: [LHC16pp_deghjop,LHC16pp_kl,LHC17pp,LHC18pp] #list of periods
+      dirmerged: /data/Derived/DskINT7withJets/vAN-20190804_ROOT6-1/pp_data/datavaltot
```

Comment:
this should be local, so without the /data/.../prefix, simply: datavaltot
(this is merged over all periods, so there is only one path needed)